src/components/views/messages/MessageActionBar.js

```diff
@@ -35,48 +37,76 @@ export default class MessageActionBar extends React.PureComponent {
         onFocusChange: PropTypes.func,
     };
 
+    state = {
```

Comment:
let's set this in the constructor with `this.state` please (for consistency with the rest of the project)